packages/http-client/src/index.ts

```diff
@@ -1,18 +1,19 @@
 import fetch from "isomorphic-fetch";
 import type { Includes, Project } from "./index.d";
 
-const protocol = process.env.NODE_ENV === "production" ? "https" : "http";
-
 export const loadProject = async ({
+  webstudioAPIUrl = null,
   projectId,
-  host = process.env.DESIGNER_HOST || "localhost:3000",
   include = { tree: true, props: true, breakpoints: true },
 }: {
+  webstudioAPIUrl: string | null;
```

Comment:
1. there is no non-webstudio api url right?
2. if its optional, use optional type, almost never null, we are using null type basically when we have to deal with dom apis which use null, mostly I am trying to hide null in the abstraction and never expose it